test/Microsoft.AspNetCore.Routing.Tests/RouteValueEqualityComparerTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Xunit;
```

Comment:
Add the copyright header used everywhere to the new file

Reply:
Done